src/components/EventList/index.js

```diff
@@ -4,10 +4,32 @@ import PropTypes from 'prop-types';
 import moment from 'moment';
 import Table from '../Table';
 
+const dateTimeSort = (dateTimeA, dateTimeB) => {
+  return moment(dateTimeA).diff(moment(dateTimeB));
+};
+
 const columns = [
   { title: 'Event Name', field: 'title' },
-  { title: 'Start Time', field: 'startDateString' },
-  { title: 'End Time', field: 'endDateString' },
+  {
+    title: 'Start Time',
+    field: 'startDateString',
+    customSort: (eventA, eventB) => {
+      const { startDate: startDateTimeA } = eventA;
```

Comment:
pull out the (eventA, eventB) => into it's own function so you don't have to redeclare it in customSort for both Start Time and End Time

Reply:
As in put (eventA, eventB) => into dateTimeSort and modify dateTimeSort a bit so it handles both start time and end time?

Reply:
sure - my reasoning was that dateTimeSort is essentially a comparator (and perhaps the Sort is a misnomer here) and so customSort should take in a comparator between events

Reply:
so one way of cleaning this up would be to write a single comparator that just compares events by comparing the two datetime objects